### PR TITLE
print HostPathType for kubectl describe

### PR DESCRIPTION
**Pull request by dixudx**
**What this PR does / why we need it**:
Since #46597 has already added `HostPathType` for `HostPath`, we should print it when we describe it.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
@thockin PTAL
/assign @smarterclayton @liggitt @pwittrock

**Release note**:

```release-note
None
```

### CODE REVIEW

File: pkg/printers/internalversion/describe.go [hunk 1]
```diff
@@ -770,8 +770,14 @@ func describeVolumes(volumes []api.Volume, w PrefixWriter, space string) {
 }
 
 func printHostPathVolumeSource(hostPath *api.HostPathVolumeSource, w PrefixWriter) {
+	hostPathType := "Not set yet"
```
Comment (liggitt) [inline pkg/printers/internalversion/describe.go]: `<none>` is more standard, I think

**Comment by liggitt** [comment]
@kubernetes/sig-cli-pr-reviews

File: pkg/printers/internalversion/describe.go (hunk 1 above)
Comment (dixudx) [reply pkg/printers/internalversion/describe.go]: Done. PTAL. Thanks.

**Comment by dixudx** [comment]
@liggitt Need your milestone label to get it merged. Thanks.

**Pull request merged**